ansible/roles/test/files/ptftests/voq.py

```diff
@@ -386,6 +387,9 @@ def check_ip_mtu(self):
 
             masked_exp_pkt = Mask(exp_pkt)
             masked_exp_pkt.set_do_not_care_scapy(scapy.Ether, "dst")
+            if self.ignore_ttl:
```

Comment:
Why we need to ignore_ttl in VoQ chassis ?

Reply:
This gets set when we are on VS and the behavior is different than with HW.